src/web-components/nav-menu/nav-menu.js

```diff
@@ -1,11 +1,97 @@
 import html from './nav-menu.html';
 import style from './nav-menu.sass';
+import menuItems from './menu-items.json';
 
 const template = document.createElement('template');
+
+const componentStyle = `
```

Comment:
Heb je de styling zo gedaan omdat het nog niet gelukt is met het importeren van de styles?